compass/ocean/tests/nonhydro/solitary_wave/streams.forward

```diff
@@ -0,0 +1,30 @@
+<streams>
+
+<immutable_stream name="mesh"
+                  filename_template="init.nc"/>
+
+<immutable_stream name="input"
+                  filename_template="init.nc"/>
+
+<immutable_stream name="restart"/>
+
+<stream name="output"
+        type="output"
+        filename_template="output.nc"
+        output_interval="0000_01:30:00"
+        clobber_mode="truncate">
+
+    <var_struct name="tracers"/>
+    <var name="xtime"/>
+    <var name="normalVelocity"/>
+    <var name="layerThickness"/>
+    <var name="pressure"/>
+    <var name="density"/>
+    <var name="nonhydrostaticPressure"/>
+    <var name="verticalVelocity"/>
```

Comment:
```suggestion
```
These variables are not available for the hydrostatic simulation.

Reply:
I think this is fine.  These variables will be written when the test case is run in nonhydro mode and not written when running in hydro mode.  There should not be any complaint from MPAS-O.

Reply:
Ah, good point!